### PR TITLE
Gam/images

### PR DESCRIPTION
**Pull request by gamiller**
Putting images on the backend so that they can be accessed from the injection script

weee grace has a backend pull request whatup

### CODE REVIEW

File: .bash_profile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+export PATH=$PATH:/usr/local/sbin
```
Comment (zatannenbaum) [inline .bash_profile]: this file needs to be removed

Comment (zatannenbaum) [reply .bash_profile]: and the celerybeat-schedule.db file

File: hindsite/static/css/main.css [hunk 2]
```diff
@@ -46,3 +46,16 @@
 h2, p {
   font-family: 'Raleway';
 }
+
+#nav-bar-img{
+  height: 67px;
+  width: auto;
+}
```
Comment (zatannenbaum) [inline hindsite/static/css/main.css]: this is unrelated, yeah?

**Review by zatannenbaum — COMMENTED** [review]
do we need all of the images or just certain images? also, are these being served to the front end?

File: hindsite/static/css/main.css (hunk 2 above)
Comment (gamiller) [reply hindsite/static/css/main.css]: ya, idk how that got there? i didnt edit css

**Comment by zatannenbaum** [comment]
Still needs the images to be cleaned up and the celerybeat-schedule.db file removed

**Comment by gamiller** [comment]
deleted errything, and we need the other images

**Comment by zatannenbaum** [comment]
we don't need the orange background blue h logos
